src/components/common/Stepper.vue

```diff
@@ -71,8 +72,8 @@ export default class Stepper extends Vue {
       id: 'step-2-btn',
       icon: 'mdi-account-multiple-plus',
       text: 'Add People\nand Roles',
-      to: '/people-roles',
-      disabled: true,
+      to: '/add-people-roles',
```

Comment:
Please also update Actions.vue so that clicking on back / next buttons include this new page in the navigation flow.

Reply:
> Please also update Actions.vue so that clicking on back / next buttons include this new page in the navigation flow.

Lekshmi says we can implement something clever for this once all the pages are implemented :)

Reply:
K, noted and will update when all the pages are implemented!